spec/after_hook/dumpers/after_hook_dumper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FILENAME = 'test.out'
+
+ResponseDumper.define 'after_hook' do
+  after do
+    tmp_dir = Dir.glob('../../tmp_*').last
```

Comment:
Rather than rely on `last` being correct, we could use a method to reference the temporary directory.